app/javascript/packages/document-capture/services/upload.ts

```diff
@@ -126,6 +128,9 @@ const upload: UploadImplementation = async function (payload, { method = 'POST',
 
     error.isFailedResult = !!result.result_failed;
 
+    // TODO send isFailed from the BE
+    error.isFailedSelfie = true; // TODO !!result.selfie_failed;
```

Comment:
I think we want to remove these comments

```suggestion
    error.isFailedSelfie = true;
```

Reply:
Though looking at it again - I wonder if they're still there because they haven't been implemented yet. I think we probably shouldn't be just setting `true` here.

Reply:
[Fixed](https://github.com/18F/identity-idp/pull/10042/commits/ed2bd0ba7e1d663316c9c2d98ae18853871dd063).

This is implemented, but good catch: Needed to be hooked up. When you get a minute, could you repeat the testing you did? This is a big enough issue I'm going to retest too.

Reply:
Thanks for fixing it! I tested again following the testing plan and it still worked!